frontend/webpack.common.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src/index.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'public'),
+    },
+
+    module: {
+        rules: [
+            {
+                test: /\.jsx?$/,
```

Comment:
You need to integrate eslint into js(x) compilation process to enforce code standards.